Resources/Prototypes/Entities/Objects/Decoration/posters.yml

```diff
@@ -0,0 +1,19 @@
+- type: entity
+  parent: BaseItem
+  id: RolledPosterContraband
+  name: rolled contraband poster
+  description: "This poster comes with its own automatic adhesive mechanism, for easy pinning to any vertical surface. Its vulgar themes have marked it as contraband aboard Nanotrasen space facilities."
+  components:
+  - type: Sprite
+    sprite: Structures/Wallmounts/posters.rsi
+    state: rolled_contraband
+
+- type: entity
+  parent: BaseItem
+  id: RolledPosterLegit
+  name: rolled legit poster
+  description: "An official Nanotrasen-issued poster to foster a compliant and obedient workforce. It comes with state-of-the-art adhesive backing, for easy pinning to any vertical surface."
+  components:
+  - type: Sprite
+    sprite: Structures/Wallmounts/posters.rsi
+    state: rolled_legit
```

Comment:
afaict these are just placebo posters, which you cant actually place on walls. we decided on not adding non-functional items into the game, so these would either need to be removed or coded to actually be placeable